rust/connlib/clients/apple/build-rust.sh

```diff
@@ -38,22 +66,27 @@ else
   fi
 fi
 
+MESSAGE="Building Connlib"
+
 if [[ -n "$CONNLIB_MOCK" ]]; then
-  LIPO_ARGS="--features mock"
+  MESSAGE="${MESSAGE} (mock)"
+  FEATURE_ARGS="--features mock"
 fi
 
-# if [ $ENABLE_PREVIEWS == "NO" ]; then
-
-  if [[ $CONFIGURATION == "Release" ]]; then
-      echo "BUILDING FOR RELEASE ($TARGETS)"
-
-      cargo lipo --release --manifest-path ./Cargo.toml  --targets $TARGETS $LIPO_ARGS
-  else
-      echo "BUILDING FOR DEBUG ($TARGETS)"
+if [[ $CONFIGURATION == "Release" ]]; then
+  echo "${MESSAGE} for Release"
+  CONFIGURATION_ARGS="--release"
+else
+  echo "${MESSAGE} for Debug"
+fi
 
-      cargo lipo --manifest-path ./Cargo.toml  --targets $TARGETS $LIPO_ARGS
-  fi
+if [[ -n "$CONNLIB_TARGET_DIR" ]]; then
+  CONNLIB_TARGET_ARGS="--target-dir $CONNLIB_TARGET_DIR"
```

Comment:
What is the purpose of specifying `--target-dir`?

Reply:
CONNLIB_TARGET_DIR (set in Build.xcconfig) is used to set the path where Xcode should look into for the connlib static libraries. For consistency, the same variable is passed as `--target-dir` to cargo, to ask cargo to place the built objects there. (If there was a way to query the target dir from the cargo build system, that would be better.)

Reply:
No need to pass the cli arg you can use [`CARGO_TARGET_DIR`](https://doc.rust-lang.org/cargo/reference/config.html#buildtarget-dir).

Reply:
No need to pass the cli arg you can use [`CARGO_TARGET_DIR`](https://doc.rust-lang.org/cargo/reference/config.html#buildtarget-dir).

Reply:
I was using `—target-dir` instead of `CARGO_TARGET_DIR` because it might be more noticeable in the log.

I’ll change it to set `CARGO_TARGET_DIR` and print a message about it.

Reply:
Fixed in [1beeac8](https://github.com/firezone/firezone/pull/1759/commits/1beeac8e78996030347e1995ec148059fc51ba32).